Makefile

```diff
@@ -434,3 +434,13 @@ ocp-aws-credentials: ## Add CredentialsRequest for OCP on AWS
 test-e2e: ginkgo ## Run end to end (E2E) tests
 	@test -n "${KUBECONFIG}" -o -r ${HOME}/.kube/config || (echo "Failed to find kubeconfig in ~/.kube/config or no KUBECONFIG set"; exit 1)
 	$(GINKGO) -r --keep-going --require-suite --vv  ./test/e2e -coverprofile cover.out
+
+# Revert all version or build date related changes
+.PHONY: bundle-reset
+bundle-reset:
+	VERSION=0.0.1 $(MAKE) manifests bundle
+	# empty creation date
+	sed -r -i "s|createdAt: .*|createdAt: \"\"|;" ${BUNDLE_CSV}
```

Comment:
Why is it needed? Running bundle won't initialize the `createdAt` field?

Reply:
but running bundle-update would.
so this can be used to make sure we have a "clean slate".

Reply:
bundle-update is called only when building the bundle container (`bundle-build`) or setting him up for community (`bundle-community`). For both of the places, it makes sense to "update" the bundle.
>sometimes I wan't to just make sure my files are up to speed without running tests

Considering your comment below, I see your motivation for having this target